sections/03_categorize.md

```diff
@@ -238,12 +261,90 @@ Ratner, Bach, and Ré described labeled training data as "The _New_ New Oil"
 framing, data are abundant and not a scarce resource. Instead, new approaches
 to solve problems arise when labeled training data become sufficient to enable
 them. Based on our review of research on deep learning methods to categorize
-disease, the latter framing rings true. We expect each of these approaches to
+disease, the latter framing rings true.
+
+In addition to methodological improvements, a robust culture of data sharing -
```

Comment:
A robust culture of data sharing is a fairly general idea.  We can keep it here for now and consider whether to move it to the Data Limitations subsection of Discussion later (if we want to keep that subsection).